Others/Segmented Sieve.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+Programmed By Shawon
+
+This algorithm will generate prime
+numbers in a given range with less time
+complexity. In input,at first take the
+number of test-cases,then give the input.
+
```

Comment:
- Please write comments in Doxygen format. 
- Add time complexity and space complexity.